packages/ui/v2/core/WizardForm.tsx

```diff
@@ -1,7 +1,7 @@
 import { useRouter } from "next/router";
 
+import { useLocale } from "@calcom/console/modules/common/hooks/useLocale";
```

Comment:
This is to be able to use this component either from Web or Console

Reply:
Why?